source/2015-09-08-meteor-and-three-js.html.markdown

```diff
@@ -0,0 +1,230 @@
+---
+title: Meteor and THREE.js
+date: 2015-09-08 21:30
+tags:
+---
+
+We at PANTER already combined Meteor with a 2d-canvas-libraries ([Konva / Kinetic.js](https://github.com/konvajs/konva)), so 3d-rendering is the next logical step.
+
+## Introduction
+
+Meteor is a "fullstack" webframework, covering both client and server. If you haven't heard about it, check it out at [meteor.com](https://www.meteor.com/).
+
+Meteor update its views as soon as the underlying data changes in a reactive way. The default rendering engine *Blaze* and the template-language "spacebars" provide an easy way to work with html (btw. official *react*-support coming soon), but what if we want to use other renderers such as THREE.js and update a 3D-View in a reactive way?
+
+## Preparations
+
+I use the meteor-smartpackage 'limemakers:three' in this example, but you can add the THREE.js-files also manually.
```

Comment:
Above you talk about "we", you should be consistent about the roles. How about "In this example the meteor-smartpackage 'limemakers:three' is used..."